src/document.rs

```diff
@@ -220,15 +220,26 @@ impl Document {
 
     /// Metadata for generating RSS feeds
     pub fn to_rss(&self, root_url: &str) -> rss::Item {
+        let description = self.attributes
+            .get("description")
+            .or_else(|| self.attributes.get("excerpt"))
+            .or_else(|| self.attributes.get("content"))
```

Comment:
`.or_else(|| self.attributes.get("excerpt"))` is more lazy

it's hard to tell in advance which one is faster, so I'll take your approach as a more readable